lua/framework/physics/lua_material.cc

```diff
@@ -282,8 +282,7 @@ MatSetProperty(lua_State* L)
     // If the property is valid
     if (location_of_prop >= 0)
     {
-      auto prop =
-        std::static_pointer_cast<SingleStateMGXS>(cur_material->properties_[location_of_prop]);
+      auto prop = std::static_pointer_cast<MGXS>(cur_material->properties_[location_of_prop]);
 
       // Process operation
       if (operation_index == static_cast<int>(OpType::SIMPLEXS0))
```

Comment:
the name SIMPLEXS0 is cryptic. Better name ?

Reply:
I think `SIMPLEXS0` is just a pure absorber. We could change it to PURE_ABSORBER.

Reply:
Changed to PURE_ABSORBER.